integration-tests/testkit/cli.ts

```diff
@@ -35,40 +37,18 @@ async function exec(cmd: string) {
   return outout.stdout;
 }
 
-export async function schemaPublish(args: string[]) {
-  const registryAddress = await getServiceHost('server', 8082);
-  return await exec(
-    ['schema:publish', `--registry.endpoint`, `http://${registryAddress}/graphql`, ...args].join(
-      ' ',
-    ),
-  );
-}
-
-export async function schemaCheck(args: string[]) {
-  const registryAddress = await getServiceHost('server', 8082);
-
-  return await exec(
-    ['schema:check', `--registry.endpoint`, `http://${registryAddress}/graphql`, ...args].join(' '),
-  );
-}
-
-export async function schemaDelete(args: string[]) {
-  const registryAddress = await getServiceHost('server', 8082);
-
-  return await exec(
-    ['schema:delete', `--registry.endpoint`, `http://${registryAddress}/graphql`, ...args].join(
-      ' ',
-    ),
-  );
-}
-
-async function dev(args: string[]) {
-  const registryAddress = await getServiceHost('server', 8082);
-
-  return await exec(
-    ['dev', `--registry.endpoint`, `http://${registryAddress}/graphql`, ...args].join(' '),
-  );
-}
+export const hiveCLI = HiveCLI.create({
```

Comment:
@jasonkuhrt  Can we please keep this out of the scope of this PR, I feel like it is getting diverged from the original intent.

Reply:
I can definitely understand that and was partly expecting a comment like this :). In my defense, the inferred library is arguably in scope in the sense that it provides the base to rapidly test each command, many of which are currently not tested.

There are a few approaches I could take here:

1. Test JSON using the existing pattern of re-creating wrappers for each command.
2. Test JSON using the inferred library, but do not update any existing tests to use that library yet.
3. Do not test JSON on any commands beyond those that are currently tested.

Please tell me which you prefer! 🙏

Reply:
For now, lets use the existing method of running the commands. We don't need to extensively test every single command.

Testing the schema publish, schema check and app publish flow is crucial and we can have json tests for these.